docs/install/upgrade.md

```diff
@@ -0,0 +1,39 @@
+# Upgrade
+
+This article walks you through how to upgrade your Coder server.
+
+To upgrade your Coder server, simply reinstall Coder using your original method
+of [install](../install.md).
+
+<blockquote class="danger">
+  <p>
+  Prior to upgrading Coder, we _highly_ recommend taking a database snapshot, as
```

Comment:
```suggestion
  Prior to upgrading a production Coder deployment, take a database snapshot since Coder doesn't support rollbacks.
```

Due to the severe irreversibility of database corruption, we need to take a firm stance here.